app/_hub/kong-inc/datadog/index.md

```diff
@@ -75,7 +75,7 @@ params:
       description: The port to send data to on the upstream server
     - name: metrics
       required: true
-      default: All metrics are logged
+      default: 
       datatype: array of record elements
       description: |
         List of Metrics to be logged. Available values are described at [Metrics](#metrics).
```

Comment:
If you're leaving out the default (and I understand why, since the default is "everything"), then it might be useful to note in the description here that all metrics are logged by default, unless you specify a list explicitly (if I'm understanding correctly). So, something like:

> By default, the plugin logs all available metrics. If you specify an array of metrics, only the listed metrics are logged.

Need dev confirmation that that's correct though.

Reply:
@svenwal What do you think about Lena's comment above?